### PR TITLE
Set Index HTML as Homepage

### LINKED ISSUE #28: Create and serve index.html
## Main Goal
Have your app default an actual webpage

### When Accomplished
- [x] Hitting `...:1820/` loads `index.html` rather than the directory itself
- [x] `index.html` *at very least* contains the ability to navigation to each view
- [x] Navigation *to* any view *from* any view is possible in app

### Notes
- Other than navigation, I don't care what your "home page" shows. You can make something up or choose a view you've already made.
- This is something I spaced on that should have been done when you initially set up the server (#4). My bad.

### PR DESCRIPTION
**Pull request by Jropp**
## Changes

1. Created index.html in root of project. Python server automatically detects it as entry point. 
2. Added 'home' page to nav bar
3. Added derp cat photo

## Issue 

Closes #28

### CODE REVIEW

**Comment by Jropp** [comment]
@collincahill try it now. Note the change in how to get things started in the README

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <!-- TODO: enter site and company title -->
+    <title>New User</title>
```
Comment (kddrake) [inline src/index.html]: Might need changed seeing as this isn't the new user page

**Comment by Jropp** [comment]
- [ ] serve up src and lands at index.html

- [ ] update yarn lock

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 node_modules/
 package-lock.json
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+yarn.lock
```
Comment (kddrake) [inline .gitignore]: Sorry, by "update" I meant actually push changes to `yarn.lock` up. You'll want `yarn.lock` apart of your repository, it'll keep dependencies locked in for anyone who's using your project.

File: README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ This app allows you to create and edit user profiles, each of which have an ID,
 ### NPM Scripts
     To serve the app (also runs ux-lint):
 
-        `$ yarn start`
+        Run `$ yarn start` then open https://localhost:1820/src/ in the browser.
```
Comment (kddrake) [inline README.md]: Make sure to update this to just `https://localhost:1820`

File: src/edit-user.html [hunk 4]
```diff
@@ -7,12 +7,13 @@
     <!-- TODO: enter site and company title -->
     <title>New User</title>
```
Comment (collincahill) [inline src/edit-user.html]: This ought to be 'Edit User'

**Comment by Jropp** [comment]
@collincahill @kddrake issues addressed

**Comment by collincahill** [comment]
Looks great 😀 Merging.

**Pull request merged**